server/repository/src/migrations/v2_11_00/rename_authorised_on_purchase_order_status_enum.rs

```diff
@@ -0,0 +1,29 @@
+use crate::migrations::*;
+
+pub(crate) struct Migrate;
+
+impl MigrationFragment for Migrate {
+    fn identifier(&self) -> &'static str {
+        "rename_authorised_on_purchase_order_status_enum"
```

Comment:
nit can't this all be done in one migration? It's confusing to have multiple migrations for the same time in different files